examples/ending_locations/subscript.py

```diff
@@ -16,8 +17,7 @@
 x = m[  :  ]
 
 
-from typing import List
 
```

Comment:
now that there's no separating import, ensure there's exactly two blank lines separating the above code with the function definition below